### PR TITLE
Consistent iconography between docs and platform

### PR DESCRIPTION
**Pull request by gordonhart**
This PR brings in various [Blueprint](https://github.com/palantir/blueprint) icons as SVG for usage in documentation. This setup allows us to have consistent iconography between these docs and our app.

Fixes KOL-2494
Fixes PRO-125

### CODE REVIEW

**Comment by gordonhart** [comment]
Note that the "Packaging for Automated Evaluation" icon is still the [container Octicon](https://primer.style/design/foundations/icons/container-24). I haven't found a suitable replacement from Blueprint, and it doesn't look too out-of-place.

File: docs/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-icon: octicons/beaker-24
+icon: kolena/test-suite-16
```
Comment (ly-kolena) [inline docs/index.md:2]: Is it intentional to use the same icon for test-suite for `getting-started` in the side-bar?

**Review by ly-kolena — APPROVED** [review]

File: mkdocs.yml [hunk 2]
```diff
@@ -10,9 +10,14 @@ edit_uri: edit/trunk/docs/
...
-      - installing-kolena.md
-      - building-a-workflow.md
+      - Quickstart: quickstart.md
+      - Installing <code>kolena</code>: installing-kolena.md
+      - Building a Workflow: building-a-workflow.md
+      - Core Concepts:
+          - core-concepts/index.md
+          - Workflow: core-concepts/workflow.md
+          - Test Case & Test Suite: core-concepts/test-suite.md
+          - Model: core-concepts/model.md
```
Comment (munkyshi) [inline mkdocs.yml:20]: Flagging that it doesn't feel like `workflow`, `Test Case & Test Suite`, and `Model` are children of `Core Concepts` -- wonder if there's any way to indent them.
Screenshot 2023-06-12 at 17 20 42

**Review by munkyshi — COMMENTED** [review]
At the bottom of http://localhost:8999/building-a-workflow/:
Screenshot 2023-06-12 at 17 22 08

Same with the bottom of http://localhost:8999/core-concepts/model/
Screenshot 2023-06-12 at 17 24 29

As well as the links at the bottom of:
- http://localhost:8999/advanced-usage/packaging-for-automated-evaluation/
- http://localhost:8999/metrics-guide/
- http://localhost:8999/reference/errors/
- http://localhost:8999/reference/workflow/metrics/
- http://localhost:8999/reference/legacy/built-in/base/

File: docs/index.md (hunk 1 above)
Comment (munkyshi) [reply docs/index.md:2]: 👍 would be good to use a different icon

Comment (gordonhart) [reply docs/index.md:2]: Will address in a follow-up

File: mkdocs.yml (hunk 2 above)
Comment (gordonhart) [reply mkdocs.yml:20]: Addressing in a follow-up. The content isn't quite ready yet

**Pull request merged**